app/controllers/users.js

```diff
@@ -351,3 +362,15 @@ exports.invitePlayersByMail = function (req, res) {
     });
   });
 };
+
+/** Gathers User Info and Games Logs */
+exports.game = function (req, res) {
+  Game.find().exec((err, games) => {
+    if (err) return err;
+    if (!games) return new Error('Failed to load Game');
+    return res.status(200).json({
+      data: games,
```

Comment:
Is this returning all the games or just the games for the user that is logged in?